ImmuDB4Net/ImmuStateSerde.cs

```diff
@@ -35,26 +34,13 @@ public class ImmuStateSerde
         if (session == null)
             return null;
         string contents = File.ReadAllText(fileName);
-        var deserialized = JsonSerializer.Deserialize<Dictionary<string, ImmuState>>(contents);
-        statesMap.Clear();
-        if (deserialized == null)
-        {
-            return null;
-        }
-        deserialized.ToList().ForEach(pair => statesMap.Add(pair.Key, pair.Value));
-        string key = session.ServerUUID + "_" + database;
-        if (statesMap.TryGetValue(key, out var state))
-        {
-            return state;
-        }
-        return null;
+        return JsonSerializer.Deserialize<ImmuState>(contents);
     }
 
     public void Write(string fileName, Session session, ImmuState state)
```

Comment:
`session` parameter should be removed

Reply:
will remove the class entirely, as you suggested. I also think it's better, in fact have already discussed about it.